src/routes/isbn-registry/publishers.js

```diff
@@ -112,11 +112,18 @@ export default function (permissionMiddleware) {
 
   async function downloadEmailList(req, res, next) {
```

Comment:
getEmailList?

Reply:
As this is meant to be a temporary solution, the naming convention is already directed towards the use case for sending group emails through the system instead of other means.